src/sentry/issues/attributes.py

```diff
@@ -86,8 +88,24 @@ def send_snapshot_values(
 
 
 def produce_snapshot_to_kafka(snapshot: GroupAttributesSnapshot) -> None:
-    payload = KafkaPayload(None, json.dumps(snapshot).encode("utf-8"), [])
-    _attribute_snapshot_producer.produce(Topic(settings.KAFKA_GROUP_ATTRIBUTES), payload)
+    if settings.SENTRY_EVENTSTREAM != "sentry.eventstream.kafka.KafkaEventStream":
+        try:
+            resp = requests.post(
+                f"{settings.SENTRY_SNUBA}/tests/entities/group_attributes/insert",
+                data=json.dumps([snapshot]),
+            )
+            if resp.status_code != 200:
+                raise snuba.SnubaError(
+                    f"HTTP {resp.status_code} response from Snuba! {resp.content}"
+                )
```

Comment:
Doesn't it ever respond with `204` and such?

Reply:
This endpoint is pretty simple, since it's just a test endpoint that allows us to sidestep using the Kafka topic in tests: https://github.com/getsentry/snuba/blob/3e1fbb5d300cd7d19b4c4dcbd47b495bd3d32a4a/snuba/web/views.py#L651